### ISSUE TITLE
Exception when using ModelCheckPoint after model.optimizer.lr = 1e-5

### ISSUE THREAD

**Issue by sravya8** [fchollet/keras#5218]
I am able to use ModelCheckPoints, a nice feature to save weights based on defined metrics. But I am hitting an exception if used after setting the learning rate. 
"Exception: 'get_value() can only be called on a variable. If you have an expression instead, use eval()" 
Attaching the gists for Stack trace and reproduction.

Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [ x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [x ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

Link to reproduction:https://gist.github.com/sravya8/c62edad75eb4e87021d37f7841172611
Link to stack trace: https://gist.github.com/sravya8/ec21fa5e6bf94925d916502ac9d23650

**Comment by bstriner** [comment]
Learning rate is a variable not a float. This is how to change the LR correctly.

```python
K.set_value(model.optimizer.lr, 1e-5)
```

Cheers,
Ben

**Comment by sravya8** [comment]
Perfect! That works, thanks!

**Comment by mayankpj** [comment]
thanks bstringer for resolving this 👍 
Thanks srvya for bringing this up, I noticed your nice work elsewhere in courses as well!
Good luck.
